src/engine/SCons/Environment.xml

```diff
@@ -1308,6 +1306,28 @@ env.Depends(bar, installed_lib)
 </summary>
 </scons_function>
 
+<scons_function name="Detect">
+<arguments signature="env">
+(progs)
+</arguments>
+<summary>
+<para>
+Returns the first value from <parameter>progs</parameter>
+(which may be a string or a list of strings)
+which is found by looking in the paths specified
+by <envar>PATH</envar> in the <envar>ENV</envar>
```

Comment:
I think better to just use ```env['ENV']['PATH']``` and ```env['ENV']['PATHEXT']``` than the explain it in words..

Also this is really just a wrapper around WhereIs()... so maybe just state that this is a wrapper around WhereIs() which will iterate over a single or list of programs being searched for..

Reply:
Made these changes.  Left the wrapper comment simple unamplified, since the return type is different (name from list vs path) and since Detect doesn't take extra args.  Also ended up fiddling WhereIs wording.